library/alloc/src/collections/btree/node.rs

```diff
@@ -1739,9 +1725,35 @@ unsafe fn slice_remove<T>(slice: &mut [MaybeUninit<T>], idx: usize) -> T {
         let slice_ptr = slice.as_mut_ptr();
         let ret = (*slice_ptr.add(idx)).assume_init_read();
         ptr::copy(slice_ptr.add(idx + 1), slice_ptr.add(idx), len - idx - 1);
+        if cfg!(debug_assertions) {
+            // Tell Miri about the trailing element, and zero it in debug builds.
```

Comment:
What does it mean to "tell Miri" about an element?

Reply:
As in the description, that is became uninitialized.

Reply:
Ah I see... that wasn't clear (also because it's in the hidden part of the diff).